ocaml/xapi/xapi_xenops.ml

```diff
@@ -671,6 +671,34 @@ module MD = struct
         )
         [] vm.API.vM_VGPUs
 
+  let of_vusb ~__context ~vm ~pusb =
+    let open Vusb in
+    try
+      let path = pusb.API.pUSB_path in
+      let pathList= Xstringext.String.split '-' path in
+      let hostbus = List.nth pathList 0 in
+      let hostport = List.nth pathList 1 in
+      (** Here version can be 1.10/2.00/3.00, so just get the first character *)
+      let version = List.hd (Xstringext.String.split '.' pusb.API.pUSB_version) in
```

Comment:
I don't think this is necessary - just send down the whole string to xenopsd (e.g. "3.00")

Reply:
yes 👍

Reply:
updated.